fractions/microprofile/microprofile-fault-tolerance/src/main/java/org/wildfly/swarm/microprofile/faulttolerance/MicroProfileFaultToleranceFraction.java

```diff
@@ -39,4 +40,5 @@ public boolean isSynchronousCircuitBreakerEnabled() {
     @AttributeDocumentation("Enable/disable synchronous circuit breaker functionality. If disabled, `CircuitBreaker#successThreshold()` of value greater than 1 is not supported. Moreover, circuit breaker does not necessarily transition from `CLOSED` to `OPEN` immediately when a fault tolerance operation completes. However, applications are encouraged to disable this feature on high-volume circuits.")
     @Configurable("swarm.microprofile.fault-tolerance.synchronous-circuit-breaker")
     private Defaultable<Boolean> synchronousCircuitBreaker = Defaultable.bool(true);
```

Comment:
Is the config option used somewhere?

Reply:
Excellent question. After checking the answer is "no" :-(. Do you have a suggestion to make it usable? I think about creating a configuration bean in smallrye ft that we would specialize here... Do you have a better idea?

Reply:
Yes, you could either provide a bean which provides additional configuration or even try to implement an MP `ConfigSource`?

Reply:
I added a new ConfigSource to read config from Thorntail but it is ignored. Any clue @mkouba ?